src/codeflare_sdk/utils/demos.py

```diff
@@ -0,0 +1,23 @@
+import pathlib
+import shutil
+
+package_dir = pathlib.Path(__file__).parent.parent.resolve()
+demo_dir = f"{package_dir}/demo-notebooks"
+
+
+def clone_demo_nbs(dir: str = "./demo-notebooks"):
```

Comment:
Just a thought what if there would be an `overwrite` arg  allowing the directory to be overwritten?